backend-api/tests/testUtils/matchers.ts

```diff
@@ -36,12 +36,34 @@ function deepEquals(subject: unknown, target: unknown): boolean {
   return JSON.stringify(subject) === JSON.stringify(target);
 }
 
+const toBeValidUuid = function toBeValidUuid(candidate: unknown) {
```

Comment:
What is the value of this wrapper opposed to just using the isValidUuid function you've created in the test?

Reply:
Just being able to use it as a Jest matcher, really. It's a nice bit of sugar to be able to say `expect(value).toBeValidUuid()` or, as we're doing, using an embedded 'asymmetric matcher' to check that a specific field of JSON is a valid UUID. Rather than pulling out the field and checking the value of a boolean function. 

I probably wouldn't have bothered if there wasn't a read-made implementation to copy-paste over.

Reply:
Also is this just a different syntax I've not seen before or could this line be

```suggestion
const toBeValidUuid = (candidate: unknown) {
```

Ignore me if I'm just missing something obvious

Reply:
Good spot, not sure why this was done in the original implementation but as far as I can tell there's no functional difference in this case. Changed.